integration-commons/src/test/java/org/symphonyoss/integration/exception/config/NotFoundExceptionTest.java

```diff
@@ -0,0 +1,47 @@
+package org.symphonyoss.integration.exception.config;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Unit tests for {@link NotFoundException}.
+ * Created by crepache on 23/06/17.
+ */
+public class NotFoundExceptionTest {
+
+  @Test
+  public void testForbiddenUserException() {
+    NotFoundException exception = new NotFoundException();
+    String resultMessage = exception.getMessage();
+    String expectedMessage = "\n" +
```

Comment:
You could create a private method to build these String messages and avoid repeated code.